<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chrono</class>
 <widget class="QWidget" name="chrono">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>129</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chronom√®tre</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="startStop">
       <property name="text">
        <string>Start/Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reset">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="quit">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLCDNumber" name="affichageM"/>
     </item>
     <item>
      <widget class="QLCDNumber" name="affichageS"/>
     </item>
     <item>
      <widget class="QLCDNumber" name="affichageMs"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quit</sender>
   <signal>clicked()</signal>
   <receiver>chrono</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>436</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
